governance/mixed.go

```diff
@@ -48,8 +47,8 @@ type MixedEngine struct {
 	db database.DBManager
 
 	// Subordinate engines
-	// TODO: Add ContractEngine
-	headerGov *Governance
+	contractGov *ContractEngine
+	headerGov   *Governance
```

Comment:
It would be much helpful if you add description about contract and headerGov.. such as how they activate or de-activate.

Reply:
@yoomee1313 Updated in 7119926f. PTAL